core/app/core/src/lib/fields/field-logic/update-field/update-field.action.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @author SalesAgility <info@salesagility.com>.
+ */
+
+import {isEmpty} from 'lodash-es';
```

Comment:
Hi @mpuyosa91 we are missing the license here.

Reply:
Done